### PR TITLE
Refactored `get_panel()` method, and replaced `GetJudge` class

### PR DESCRIPTION
**Pull request by brett-doffing**
This PR improves the accuracy of the get_panel() method, as can be seen in the notebook found at [PR #103](https://github.com/Lambda-School-Labs/human-rights-first-asylum-ds-a/pull/103), and also replaces the GetJudge class with a static list of judges, recommended to be dealt with further by the backend.

### CODE REVIEW

File: app/ocr.py [hunk 1]
```diff
@@ -189,22 +206,28 @@ def get_date(self) -> str:
         return '{}-{}-{}'.format(sorted_dates[0].year, sorted_dates[0].month, sorted_dates[0].day)
 
     def get_panel(self) -> str:
```
Comment (Daniel-Fernandez-951) [inline app/ocr.py:208]: Output from this function should be changed to a list of strings

File: app/ocr.py [hunk 2]
```diff
@@ -189,22 +206,28 @@ def get_date(self) -> str:
...
+
+         matches = matcher(self.doc) # Run the phrase matcher over the doc
+         case_panel_members = set()
+
+         if len(matches) > 0: # If a judge from our list is found in the document
+             for match_id, start, end in matches:
+                 judge = self.doc[start:end] # A span object of the match, fetched with its `start` and `end` indecies within the doc
+                 case_panel_members.add(judge.text) # Extract the text from the span object, and add to the set of panel members
+
+         return '; '.join(case_panel_members)
```
Comment (Daniel-Fernandez-951) [inline app/ocr.py]: Output the panel members as a list here.
```suggestion
         return list(case_panel_members)
```

File: app/ocr.py (hunk 1 above)
Comment (Daniel-Fernandez-951) [inline app/ocr.py]: Function should output a list of strings.
```suggestion
    def get_panel(self) -> List[str]:
```

**Review by Daniel-Fernandez-951 — CHANGES_REQUESTED** [review]
Great improvement and thank you for starting us down the path of not relying on scraping to verify names. My only minuscule request is to output a list to make it easier for FE and BE to parse the results. Nice work!

File: app/ocr.py (hunk 2 above)
Comment (brett-doffing) [reply app/ocr.py]: Yes, I actually remember having a conversation about this, where I believe I am supposed to return a list, so thank you for pointing this out!

File: app/ocr.py [hunk 3]
```diff
@@ -205,9 +205,9 @@ def get_date(self) -> str:
         #Backend requested output format: 'yyyy-mm-dd'
         return '{}-{}-{}'.format(sorted_dates[0].year, sorted_dates[0].month, sorted_dates[0].day)
```
Comment (doffing81) [inline app/ocr.py:207]: Returns a list

File: app/ocr.py (hunk 1 above)
Comment (brett-doffing) [reply app/ocr.py]: Changed in updated commit

**Review by ren-curry — APPROVED** [review]
This is an improvement to the original method for extracting judges from the case text. As it now returns a list for the BE team based on Daniel's feedback I am happy to approve the request.

**Review by Daniel-Fernandez-951 — APPROVED** [review]
Looks great! Wish this was on the main branch earlier, had some issues with the previous solution when batch processing all the PDFs; request timeout after many repeat scrapes.

**Review by JenFaith — APPROVED** [review]
This is a much better function. I like how you added in the todo as well so the next cohort will know where to pick up.

**Pull request merged**